doc/themes/scikit-learn-modern/static/css/theme.css

```diff
@@ -823,6 +823,10 @@ div.body img {
     height: unset!important; /* Needed because sphinx sets the height */
 }
 
+div.body dd > p {
```

Comment:
Your change is not located in `doc/themes/scikit-learn/static/nature.css_t` as suggested 
in https://github.com/scikit-learn/scikit-learn/issues/12941#issuecomment-452520119.

Reply:
Yeah, that previous comment sort of threw me for a loop.  

I think there was a change to which stylesheet the documentation uses since that last comment. `doc/themes/scikit-learn-modern/static/css/theme.css` was the only stylesheet that I could change that would eventually be linked with the page in question.

I did a quick search, and I think it happened in this merge: https://github.com/scikit-learn/scikit-learn/pull/14849

Reply:
yeah @thomasjpfan rewrote the frontend / css